### DISCUSSION TITLE
Add Monorepo Group Preset for OkHttp

### DISCUSSION THREAD

**Discussion by JuJup** [renovatebot/renovate#25287] [Suggest an Idea]
### Tell us more.

I grouped the monorepo [OkHttp](https://github.com/square/okhttp#okhttp) in my local config and I thought it would make sense to put this into the included presets.

```json
{
  "packageRules": [
    {
      "matchSourceUrls": [
          "https://github.com/square/okhttp"
       ],
      "groupName": "OkHttp monorepo",
      "matchUpdateTypes": [
        "digest",
        "patch",
        "minor",
        "major"
      ]
    }
  ]
}
```

**Comment by viceice** [comment]
can go into our monorepo.ts presets file

**Reply by JuJup** [reply]
Ok, so what is the further process?
Your [documentation](https://docs.renovatebot.com/config-presets/#contributing-to-presets) told me to open a discussion and not directly a PR, is that right?

**Reply by rarkins** [reply]
Please create a PR to the `monorepos.ts` file in presets. I've updated the documentation so that it will clarify that it's ok to open monorepo preset suggestions directly without discussions in future

**Reply by JuJup** [reply]
Done
https://github.com/renovatebot/renovate/pull/25297
